walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -98,7 +100,8 @@ public void eventSplitsRestriction() {
   public void everyAttendeeIsConsidered() {
```

Comment:
This, and the two test cases below for everyAttendeeIsConsideredAllDayC and everyAttendeeIsConsideredPartialDayC are theoretically the same scenarios.  It's ok to test them separately, but assumption would be if one works, all should work.